wfe/web-front-end.go

```diff
@@ -380,24 +345,21 @@ func (wfe *WebFrontEndImpl) verifyPOST(logEvent *requestEvent, request *http.Req
 	// *anyway*, so it could always lie about what key was used by faking
 	// the signature itself.
 	if len(parsedJws.Signatures) > 1 {
-		err = core.SignatureValidationError("Too many signatures in POST body")
 		wfe.stats.Inc("WFE.Errors.TooManyJWSSignaturesInPOST", 1, 1.0)
 		logEvent.AddError("too many signatures in POST body: %d", len(parsedJws.Signatures))
-		return nil, nil, reg, err
+		return nil, nil, reg, probs.Malformed("Too many signatures in POST body")
```

Comment:
It doesn't. Just wanted to understand why. And if we're removing it we should be consistent.